src/Operations.jl

```diff
@@ -1390,7 +1391,7 @@ end
 
 # ctx + pkg used to compute parent dep graph
 function sandbox(fn::Function, ctx::Context, target::PackageSpec, target_path::String,
-                 sandbox_path::String, sandbox_project_override)
+                 sandbox_path::String, sandbox_project_override; force_latest_compat::Union{Bool, Symbol}=:autodetect)
```

Comment:
Maybe it would be cleaner to just do `::Bool=Types.decide_force_latest_compat()`?

Reply:
Do you mean `::Bool=Types.is_dependabot_job()`

Reply:
Oh yeah, sorry!